src/app/components/transaction/components/stacks-transaction-icon.tsx

```diff
@@ -0,0 +1,57 @@
+import { addressToString, PayloadType, StacksTransaction } from '@stacks/transactions';
+import { DynamicColorCircle } from '@stacks/ui';
+
+import { StxIcon } from '@app/components/icons/stx-icon';
+import { Tx } from '@app/common/api/transactions';
+import { getTxSenderAddress } from '@app/store/accounts/account-activity.utils';
+
+import { TransactionTypeIcon } from './transaction-type-icon';
+import { TransactionIconWrapper } from './transaction-icon-wrapper';
+
+interface StacksTransactionIconProps {
+  transaction: StacksTransaction;
+}
+export function StacksTransactionIcon({ transaction, ...rest }: StacksTransactionIconProps) {
```

Comment:
There are no other `...rest` props

Reply:
Thanks, I'll look over all this again today too.